languages/csharp/exercises/concept/chars/.docs/hints.md

```diff
@@ -0,0 +1,34 @@
+### 1. Replace any spaces encountered with underscores
+
+[This tutorial][chars-tutorial] is useful.
+
+[Reference documentation][chars-docs] for `char`s is here.
+
+You can retrieve `char`s from a string in the same way as elements from an array.
+
+You should use a [string builder][string-builder] to build the output string.
+
+See [this][iswhitespace] for detecting spaces. Remember it is a static method.
+
+char literals are enclosed in single quotes.
+
+### 2. Replace control characters with the upper case string "CTRL"
+
+See [this][iscontrol].
+
+### 3. Convert kebab-case to camel-case
+
+See [this][toupper].
+
+### 4. Omit Greek lower case letters
+
+`char`s support equality and comparison operators
```

Comment:
```suggestion
- `char`s support the default equality and comparison operators.
```

Reply:
Just to show I'm awake.  As far as I know there are no non-default equality and comparison operators.  I think you need Scala for that.